src/services/infrastructure/url-generator/url.generator.service.ts

```
@@ -152,91 +77,32 @@ export class UrlGeneratorService {
         return `${this.endpoint_cluster}/${UrlPathBase.VIRTUAL_CONTRIBUTOR}/${vcEntityInfo.entityNameID}`;
       case ProfileType.ORGANIZATION:
-        const organizationEntityInfo = await this.getNameableEntityInfoOrFail(
-          'organization',
-          this.FIELD_PROFILE_ID,
-          profile.id
-        );
+        const organizationEntityInfo =
+          await this.getNameableEntityInfoForProfileOrFail(
+            'organization',
+            profile.id
+          );
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)